src/artm/c_interface.cc

```diff
@@ -485,6 +485,8 @@ int ArtmOverwriteTopicModel(int master_id, int length, const char* topic_model)
 }
 
 int ArtmInitializeModel(int master_id, int length, const char* init_model_args) {
+  // check if given dictionary name is appear in instance.dictionaries_
```

Comment:
Remove the comment. Methods of c_interface should only verify local consistency of protobuf messages. All other high-level preconditions should be verified by individual methods. Particularly, MasterComponetn::InitializeModel (or whatever method implements the initialization) needs to throw an error if given name is not present in the instance.dictionaries_.